File: data/data.code.lua53.server/ReqTestJsOk.lua
ReqTestJsOk = {}
setmetatable(ReqTestJsOk, {__index = _G})
local _ENV = ReqTestJsOk


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = ReqTestJsOk})
	tab_obj:init()
	return tab_obj
end

function init(self)
	self.u64 = 0
	self.i64 = 0
end


function decode(self, pack)
	self.u64 = pack:readu64()
	self.i64 = pack:readi64()
	return self
end


function getU64(self)
	return self.u64
end

function getI64(self)
	return self.i64
end

File: data/data.code.golang.server/req_test_js_ok.go
package proto

import (
	"packet"
)

type ReqTestJsOk struct {
	u64                      uint64
	i64                      int64
}

func ReqTestJsOkDecode(pack *packet.Packet) *ReqTestJsOk {
	reqTestJsOk := &ReqTestJsOk{}

	reqTestJsOk.u64 = pack.ReadUint64()
	reqTestJsOk.i64 = pack.ReadInt64()
	return reqTestJsOk
}

func (this *ReqTestJsOk) GetU64() uint64 {
	return this.u64
}

func (this *ReqTestJsOk) GetI64() int64 {
	return this.i64
}
